tests/10-deploy

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import amulet
+import unittest
+
+
+class TestDeployment(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.deployment = amulet.Deployment(series='trusty')
+        cls.deployment.add('etcd')
+        try:
+            cls.deployment.setup(timeout=900)
+            cls.deployment.sentry.wait()
+        except amulet.helpers.TimeoutError:
+            msg = "Environment wasn't stood up in time"
+            amulet.raise_status(amulet.SKIP, msg=msg)
+        except:
+            raise
+
+    def test_single_service(self):
+        status = self.deployment.sentry['etcd'][0].run('service etcd status')
+        self.assertTrue("running" in status[0])
+
+    def test_two_node_scale(self):
+        self.deployment.add_unit('etcd')
+        self.deployment.sentry.wait()
+
+        status1 = self.deployment.sentry['etcd'][0].run('service etcd status')
+        status2 = self.deployment.sentry['etcd'][1].run('service etcd status')
+        self.assertTrue("running" in status1[0])
+        self.assertTrue("running" in status2[0])
+
```

Comment:
running != healthy cluster.  You could also run a `etcdctl cluster-health` on one of the machines here to make sure the cluster is healthy. Improvements for next iteration.